OWNERS_ALIASES

```diff
@@ -0,0 +1,25 @@
+# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+
+aliases:
```

Comment:
- sig-pm-leads to be added:
  - idvoretskyi
  - apsinha
  - calebamiles

Reply:
done